tools/common.props

```diff
@@ -0,0 +1,15 @@
+<Project>
+  <PropertyGroup>
+    <VersionSuffix>$([System.DateTime]::Now.ToString(eyyMMdd-1))</VersionSuffix>
```

Comment:
Does the "e" specifier mean anything or is it just string?
I couldn't find it anywhere here:
https://msdn.microsoft.com/en-us/library/8kb3ddd4.aspx

```C#
Console.WriteLine(System.DateTime.Now.ToString("eyyMMdd-1"));
```
**Output:** e170206-1